src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+  NavLink,
+} from 'react-router-dom';
+import './styles/_main.scss';
+import Home from './components/home';
+import Details from './components/details';
+
+const App = () => (
+  <Router>
+    <NavLink className="home_btn" to="/home">
+      <h1 className="Home_Btn_text">Home</h1>
+    </NavLink>
```

Comment:
- [ ] On the navbar, you have some missing icon, even if it's not working please make sure to add those element in your app.

### Your navbar 

![Screenshot from 2021-09-08 13-14-25](https://user-images.githubusercontent.com/22434660/132467842-91f546dd-b260-43d1-b70c-c48887c7ba82.png)

### Design given

![Screenshot from 2021-09-08 13-11-35](https://user-images.githubusercontent.com/22434660/132467721-e549c5b3-7b22-4d49-8480-dde72293958f.png)